python/ray/serve/api.py

```diff
@@ -857,10 +865,17 @@ def shutdown() -> None:
     Shuts down all processes and deletes all state associated with the
     instance.
     """
-    if _global_client is None:
+
+    try:
+        client = internal_get_global_client()
+    except RayServeException:
+        logger.info(
+            "Nothing to shut down. There's no Serve application "
+            "running on this Ray cluster."
+        )
```

Comment:
this would be a breaking change, can we instead just return silently if there's nothing to shut down?

Reply:
sorry, disregard me here...